.github/workflows/tests_linux.yml

```diff
@@ -555,12 +555,12 @@ jobs:
       - name: Run PennyLane-Lightning unit tests
         if: ${{ matrix.pl_backend != 'all'}}
         run: |
-          # TODO: Remove installing pytest-xdist with release v0.36.0
-          python -m pip install pytest-xdist
+          # TODO: Remove installing pytest-xdist and pytest-custom_exit_code with release v0.36.0
+          python -m pip install pytest-xdist pytest-custom_exit_code
```

Comment:
I don't know if we should depend upon a 3rd party package that hasn't been updated in 5 years. Is there an alternative?

Reply:
This package is the recommended way to suppress ExitCode 5 by [the official pytest docs](https://docs.pytest.org/en/stable/reference/exit-codes.html). The logic is fairly simple but I didn't want to spend anytime on it as this is just temporary and is going to be removed with release v0.36

Reply:
Hmm, I'm not really convinced we should use this. However, if temporary, then can we pin the installed version for now to `0.3.0` so it is guaranteed to be the one we can currently see on PyPI, and then ensure it is removed before the release goes out.

Reply:
okay, we don't need this package as the problem is limited to the native mcm test file ;) I fixed this using if-cond and we will remove it with v0.36 release.

Reply:
https://github.com/PennyLaneAI/pennylane-lightning/actions/runs/8699050876 :green_circle: